src/osd/PrimaryLogPG.cc

```diff
@@ -6764,8 +6872,8 @@ int PrimaryLogPG::do_osd_ops(OpContext *ctx, vector<OSDOp>& ops)
 	  ctx->op_finishers[ctx->current_osd_subop_num].reset(
 	    new SetManifestFinisher(osd_op));
 	  RefCountCallback *fin = new RefCountCallback(ctx, osd_op);
-	  refcount_manifest(ctx->obc, tgt_oloc, target, SnapContext(),
-			    true, fin, src_offset);
+	  refcount_manifest(ctx->obc, ctx->obc->obs.oi.soid, tgt_oloc, target, 
+			    SnapContext(), true, fin);
```

Comment:
Any chance you could replace the boolean parameter with an enum INCREMENT_REF or DECREMENT_REF?

Reply:
Good suggestion.